main.go

```diff
@@ -81,6 +81,12 @@ func main() {
 			Usage:  "Path to a pem encoded key file.",
 			EnvVar: "BAZEL_REMOTE_TLS_KEY_FILE",
 		},
+		cli.Int64Flag{
+			Name:   "max_idle_time",
+			Value:  -1,
```

Comment:
I think we don't need to use a negative value, as `0` would do just fine no?

Reply:
yeap